src/key/helper.js

```diff
@@ -396,11 +400,22 @@ export function isValidDecryptionKeyPacket(signature, config) {
   }
 
   if (config.allowInsecureDecryptionWithSigningKeys) {
-    // This is only relevant for RSA keys, all other signing ciphers cannot decrypt
+    // This is only relevant for RSA keys, all other signing algorithms cannot decrypt
     return true;
   }
 
   return !signature.keyFlags ||
     (signature.keyFlags[0] & enums.keyFlags.encryptCommunication) !== 0 ||
     (signature.keyFlags[0] & enums.keyFlags.encryptStorage) !== 0;
 }
+
+export function assertKeyStrength(keyPacket, config) {
```

Comment:
OK, very last nitpick: how about `checkKeyStrength` - an assertion is usually something that should never fail under normal circumstances (i.e. unless there's a bug).